### ISSUE TITLE
Add used primers to the sequence editor

### ISSUE THREAD

**Issue by dgruano** [manulera/OpenCloning_backend#113]
When a new source is created from a PCR it would be nice to add the used primers as annotations to the resulting DNA sequence so that they can be spotted in the editor. What do you think?

I imagine this can be done by adding annotations to the DSeqrecord object after the assembly is performed or somewhere near this.

**Comment by manulera** [comment]
Yes, I discussed this with @BjornFJohansson. It should be possible also to add assembly scars for gibson and others. This could be an `add_feature` parameter in the assembly default false?
